questionnaire/questionnaire-service.js

```diff
@@ -635,6 +635,23 @@ function createQuestionnaireService({
         );
     }
 
+    async function getQuestionnaireIdsBySubmissionStatus(status) {
+        return db.getQuestionnaireIdsBySubmissionStatus(status);
+    }
+
+    async function postFailedSubmissions() {
+        const questionnaireIds = await getQuestionnaireIdsBySubmissionStatus('FAILED');
+        const promises = questionnaireIds.map(async id => {
```

Comment:
Is this a good name for this array 'promises'?
Something like appResubmissionState?
And curious what if anything is done with this data when it is returned?

Reply:
This part of it was just a copy and paste from the existing `submission-routes.js`. This data is returned to the cronjob that pings this endpoint. It reports the result to Prometheus, I believe it actually just needs the length of the array rather than the actual object.